File: WOT-iOS/WOTData/WOTData/model/subclasses/VehicleprofileArmorList+CoreDataProperties.m
//
//  VehicleprofileArmorList+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 8/28/18.
//  Copyright © 2018 Pavel Yeshchyk. All rights reserved.
//
//

#import "VehicleprofileArmorList+CoreDataProperties.h"

@implementation VehicleprofileArmorList (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileArmorList *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileArmorList"];
}

@dynamic hull;
@dynamic turret;
@dynamic vehicleprofile;

@end

File: WOT-iOS/WOTData/Sources/model/autogenerated/VehicleprofileArmorList+CoreDataProperties.swift
//
//  VehicleprofileArmorList+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 5/1/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import CoreData

extension VehicleprofileArmorList {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<VehicleprofileArmorList> {
        return NSFetchRequest<VehicleprofileArmorList>(entityName: "VehicleprofileArmorList")
    }

    @NSManaged public var hull: VehicleprofileArmor?
    @NSManaged public var turret: VehicleprofileArmor?
    @NSManaged public var vehicleprofile: Vehicleprofile?
}
